Containerfile.rag-gen

```diff
@@ -0,0 +1,29 @@
+FROM registry.access.redhat.com/ubi9/python-311 as lightspeed-rag-builder
+
+USER 0
+RUN dnf install -y --nodocs --setopt=keepcache=0 --setopt=tsflags=nodocs git ruby gem
+RUN pip install torch --index-url https://download.pytorch.org/whl/cpu
+RUN pip install PyYAML huggingface_hub llama_index langchain_community \
+                faiss-cpu llama-index-vector-stores-faiss sentence_transformers \
+                llama-index-embeddings-huggingface
+RUN gem install asciidoctor
+
+WORKDIR /workdir
+RUN git clone --single-branch --branch enterprise-4.14 https://github.com/openshift/openshift-docs.git
```

Comment:
should not 4.14 be more configurable?

Reply:
It should be, but it's not clear to me at this point how to handle multiple OCP versions - within this single builder image or as an external parameter and how to pass such parameter from CI.